catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java

```diff
@@ -47,11 +48,28 @@ public String generateCreateDatabaseSql(
 
   @Override
   public String generateDropDatabaseSql(String databaseName, boolean cascade) {
+    final String dropDatabaseSql = "DROP DATABASE `" + databaseName + "`";
     if (cascade) {
-      throw new UnsupportedOperationException(
-          "MySQL does not support CASCADE option for DROP DATABASE.");
+      return dropDatabaseSql;
```

Comment:
Need to handle database not empty.

Reply:
Is the condition `!cascade`?

Reply:
> Is the condition `!cascade`?

No, MySQL will automatically delete all tables in the database when it is removed.

Reply:
Why @Clearvive said that MySQL doesn't support cascading drop, have you verified with different MySQL versions, is it a consistent behavior?

Reply:
MySQL does support cascading deletion of schema, execute `drop database {database_name}` will drop all tables under it.

@Clearvive , Please check it.